### DISCUSSION TITLE
Bugs in wiki, maybe

### DISCUSSION THREAD

**Discussion by mobeicanyue** [dani-garcia/vaultwarden#4199] [Q&A]
### Deployment environment

* vaultwarden version:
1.30.1

* Install method:
docker compose
* Clients used: 
Android
* Reverse proxy and version: 

* MySQL/MariaDB or PostgreSQL version: 

* Other relevant details:

### Steps to reproduce

In this wiki 
https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page#how-to-prevent-variable-interpolation-in-docker-composeyml

You said 
```
This is not the case when using a .env file for docker-compose.yaml
As shown below. In this case just use the single $ variant.
```

Which means the .env file using single `$`

![image](https://github.com/dani-garcia/vaultwarden/assets/81098819/3f40c1df-75cb-42eb-9f94-60eecb797de0)

But as I write it, I run `docker compose config`

error occurs like 
```
WARNING: The argon2id variable is not set. Defaulting to a blank string.
WARNING: The v variable is not set. Defaulting to a blank string.
WARNING: The m variable is not set. Defaulting to a blank string.
...
```

And I use dual `$` in the .env file, error not occur when I run  `docker compose config`

**Comment by mobeicanyue** [comment]
Hello ?

**Comment by BlackDex** [comment]
That was changed during a compose update it looks like. Seems the wiki was not updated.
Did that right now, but that isn't worth an issue ;)

**Reply by mobeicanyue** [reply]
ok I can help to update the wiki, how to pr wiki?

**Reply by BlackDex** [reply]
I already updated it.

Wiki's can be edited by anyone with a github account.
